lib/Mojolicious/Static.pm

```diff
@@ -242,6 +244,7 @@ traversing to parent directories.
 =head2 is_fresh
 
   my $bool = $static->is_fresh(Mojolicious::Controller->new, {etag => 'abc'});
+  my $bool = $static->is_fresh(Mojolicious::Controller->new, {etag => 'W/"def"'});
```

Comment:
This line is too long.

Reply:
How shall it be changed?  perhaps, this, which is what perltidy would do:

```
  my $bool
    = $static->is_fresh(Mojolicious::Controller->new, {etag => 'W/"def"'});                         
```

Should I change it now, or wait until after voting? new commit, or amend the existing one?